robot/marker_setup/markers.py

```diff
@@ -46,68 +46,65 @@ def __repr__(self) -> str:
+            return tuple(reversed(self.team_marker_colors[self.owning_team])) # Picks the team colour from above
+
+class ARENA_MARKER(BASE_MARKER): # Not much going on here. This represents a wall.
     def __init__(self, id: int) -> None:
         super().__init__(id, MARKER_TYPE.ARENA)
 
     def __repr__(self) -> str:
         return f"<Marker(ARENA)/>"
 
 
-class POTATO_MARKER(BASE_MARKER):
+class POTATO_MARKER(BASE_MARKER): # This is a game object rather than a wall. Add properties you want to keep track of
     def __init__(
-        # my_team: typing.Union[TEAM, None], wool_type: WOOL_TYPE
-        self, id: int, owner: TEAM,
+        self, id: int, owner: TEAM
     ) -> None:
-        super().__init__(
-            id,
-            MARKER_TYPE.POTATO,
-        )
+        super().__init__(id, MARKER_TYPE.POTATO)
         self.owning_team = owner
 
-        # self.wool_type = wool_type
-
     def __repr__(self) -> str:
         return f"<Marker(POTATO) owning_team={self.owning_team} />"
 
 
-class MARKER(BASE_MARKER):
+class MARKER(BASE_MARKER): # This is literally just how the code gets the different marker types.
     @staticmethod
-    def by_id(id: int, team: typing.Union[TEAM, None] = None) -> BASE_MARKER:
+    def by_id(id: int, team: typing.Union[TEAM, None] = None) -> BASE_MARKER: # team is currently unused, but it is referenced throughout the code. It is the team of the robot I believe (check this)
         """
         Get a marker object from an id
 
-        Marker IDs are between 0 and 99(? potentially higher)
-        Low marker IDs (0-39) are potatoes. The first 4 potato markers are Jacket
+        Marker IDs are greater than 0, and usually won't go higher than 200. They are 
+        read as April tags, so use an online 6x6 April tag generator to test values, 
+        such as:
+        https://chaitanyantr.github.io/apriltag (remember to set to tag36h11 for this one)
+
+        In 2024 low marker IDs (0-39) are potatoes. The first 4 potato markers are Jacket 
         Potatoes and belong to the team with the corresponding ID.
         The rest of the low markers are unowned - their owning_team property is None
         and their owner is ARENA.
 
+        It is probably recommendable that you have duplicate marker IDs, so that any damaged
+        marker can be replaced by one with equivalent game meaning - in 2024 we made ID 0 and 
+        ID 20 equivalent.
+
         Higher marker IDs (40+) will be part of the arena. These markers will be
         spaced as in 2021-2022's competition (6 markers on each side of the Arena,
         the first 50cm away from the wall and each subsequent marker 1m away from
         there). In practice these markers start at 100.
-
-        If no team is provided, all team-owned markers will be assumed to belong to
-        ANOTHER_TEAM
         """
 
-        if id >= 40:
+        if id >= 40: # If its part of the walls
```

Comment:
I like the docstring (multiline function) comment at the start of this function, and I think that it makes this comment here duplicate information as you already say:
> Higher marker IDs (40+) will be part of the arena

One way to make the code more readable is to use variables to hold magic numbers like this `40`. For example you could do:
```py
AREA_ID_LOWER_BOUND = 40
if id >= AREA_ID_LOWER_BOUND:
```